app/models/manageiq/providers/amazon/cloud_manager/virtual_template.rb

```diff
@@ -1,3 +1,4 @@
 class ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate < ::ManageIQ::Providers::CloudManager::VirtualTemplate
-  validates :cloud_subnet_id, :availability_zone_id, :ems_ref, :presence => true
+  validates :cloud_network_id, :availability_zone_id,
+            :ems_ref, :ems_id, :flavor_id, presence: true
```

Comment:
oh boo hash rockets. I had the hash rocket, but local rubocop changed it and I got excited that this repo wasn't going to use them hah but I see there's no `rubocop.yml` so it makes sense now.